README.md

```diff
@@ -25,16 +25,17 @@ Each of these patterns creates the following infrastructure:
 
 Each pattern creates the following infrastructure on the VPC:
 
+- The QuickStart VSI pattern deploys edge VPC with one VSI and a jump server VSI in the management VPC
```

Comment:
I would move this to the 2nd bullet point (so the standard VSI comes after quickstart vsi)